client/src/pages/ProfileList/List.tsx

```diff
@@ -54,6 +61,11 @@ export default function List(): JSX.Element {
     }
     fetchProfile();
   }, []);
+  const [searchTerm, setSearchTerm] = useState('');
+  const handleChange = (event: any) => {
+    setSearchTerm(event.target.value);
+    event.preventDefault();
+  };
```

Comment:
Please avoid using `any` as a type 👍🏼 You should be able to use `React.ChangeEvent` in this case.

Reply:
I'll take some time to improve my understanding of typescript, thanks!